packages/components/src/components/input/input.tsx

```diff
@@ -125,7 +125,7 @@ const Input = React.forwardRef<HTMLInputElement & HTMLTextAreaElement, TInputPro
                     })}
                 >
                     <div
-                        className={classNames('dc-input__container', className, {
+                        className={classNames('dc-input__container', {
```

Comment:
We shouldn't be doing this imo. Not really sure what the side effect could be. We need to find the route cause instead.

Reply:
Checked with @arshad-rao-deriv about this. Looks like `className` was added in feature branch and it's not there in master.